aspnet/migration/mvc.rst

```diff
@@ -219,12 +226,40 @@ The updated _Layout.cshtml file is shown below:
 
 View the site in the browser. It should now load correctly, with the expected styles in place.
 
+- Optional: You might want to try using the new layout file. For this project you can copy the layout file from the *FullAspNetCore* project. The new layout file uses :doc:`Tag Helpers </mvc/views/tag-helpers/index>` and has other improvements.
+
 Configure Bundling
 ------------------
 
 The ASP.NET MVC starter web template utilized the ASP.NET Web Optimization for bundling. In ASP.NET Core, this functionality is performed as part of the build process using `Gulp <http://gulpjs.com/>`__. We've previously configured bundling and minification; all that's left is to change the references to Bootstrap, jQuery and other assets to use the bundled and minified versions. You can see how this is done in the layout file (*Views/Shared/_Layout.cshtml*) of the full template project. See :doc:`/client-side/bundling-and-minification` for more information.
 
+Solving HTTP 500 errors
+-------------------------
+
+There are many problems that can cause a HTTP 500 error message that contain no information on the source of the problem. For example, if the *Views/_ViewImports.cshtml* file contains a namespace that doesn't exist in your project, you'll get a HTTP 500 error. To get a detailed error message, add the following code:
```

Comment:
Provide a link to the [Error Handling](https://docs.asp.net/en/latest/fundamentals/error-handling.html) article for more details.